.gitignore

```diff
@@ -7,4 +7,4 @@ npm-debug.log
 artifacts.log
 clovis-trace.log
 network-dependency-graph.png
-
+dependency-graph.json
```

Comment:
not specific to this PR, but I'm going to want a `npm run testclean` or something pretty soon for all the artifacts our tests are accumulating without using `git clean` and losing anything *I* might have locally created on purpose :)